Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM redis:6.2.6
+
+# ARGs & ENVs
+ARG SCRIPTS_PATH="/opt/redis/scripts/"
+
+COPY scripts/ ${SCRIPTS_PATH}
```

Comment:
FYI, I've made sure these scripts have executable permissions ...

```console
docker run -it zappi/redis /bin/bash
root@f10c0faa81db:/data# ls -l /opt/redis/scripts/
total 36
-rwxr-xr-x 1 root root  352 Feb 21 16:30 environment.sh
-rwxr-xr-x 1 root root  593 Feb 21 16:30 lib.sh
-rwxr-xr-x 1 root root  271 Feb 21 16:25 ping_liveness_local.sh
-rwxr-xr-x 1 root root  191 Feb 21 16:26 ping_readiness_local.sh
-rwxr-xr-x 1 root root  194 Feb 21 16:26 ping_sentinel.sh
-rwxr-xr-x 1 root root  465 Feb 21 16:27 prestop_redis.sh
-rwxr-xr-x 1 root root  784 Feb 21 16:28 prestop_sentinel.sh
-rwxr-xr-x 1 root root 1046 Feb 21 16:39 start_node.sh
-rwxr-xr-x 1 root root 1119 Feb 21 16:39 start_sentinel.sh
```

Reply:
Note that `environment.sh` and `lib.sh` are opt executable since they aren't sourced.

Reply:
> Note that `environment.sh` and `lib.sh` are opt executable since they aren't sourced.

We do [source](https://stackoverflow.com/a/20094373), but it doesn't matter.

Reply:
Oh, you mean are ~opt~ _not_ executable?

Reply:
Oops. Meant they are **not executable** since they **are sourced**. 🤦🏽‍♂️ 

Late coding. 😪